docs/en/reference/architecture.rst

```diff
@@ -24,28 +24,31 @@ performance it is also recommended that you use APC with PHP.
 Doctrine ORM Packages
 -------------------
 
-Doctrine ORM is divided into three main packages.
+Doctrine ORM is divided into four main packages.
 
--  Common
--  DBAL (includes Common)
--  ORM (includes DBAL+Common)
+-  `Collections <https://www.doctrine-project.org/projects/doctrine-collections/en/stable/index.html>`_
+-  `Persistence <https://www.doctrine-project.org/projects/doctrine-persistence/en/stable/index.html>`_
+-  `DBAL <https://www.doctrine-project.org/projects/doctrine-dbal/en/stable/index.html>`_
+-  ORM (depends on DBAL+Persistence+Collections)
 
 This manual mainly covers the ORM package, sometimes touching parts
-of the underlying DBAL and Common packages. The Doctrine code base
+of the underlying DBAL and Persistence packages. The Doctrine code base
 is split in to these packages for a few reasons and they are to...
 
 
 -  ...make things more maintainable and decoupled
--  ...allow you to use the code in Doctrine Common without the ORM
-   or DBAL
+-  ...allow you to use the code in Doctrine Persistence and Collections
+  without the ORM or DBAL
 -  ...allow you to use the DBAL without the ORM
 
-The Common Package
-~~~~~~~~~~~~~~~~~~
+The Collection and Persistence Packages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The Common package contains highly reusable components that have no
-dependencies beyond the package itself (and PHP, of course). The
-root namespace of the Common package is ``Doctrine\Common``.
+The Collection and Persistence package contains highly reusable
+components that have no dependencies beyond the package itself (and PHP,
+of course). The root namespace of the Persistence package is
```

Comment:
Persistence has a dependency to the Event-Manager. I'm not sure what this paragraph was supposed to mean back then and now. That these packages are lightweight without a dependency chain?

Reply:
I guess this paragraph was true for common: at that time, event-manager was part of common as well.

Reply:
I should use the plural, which will make the sentence "beyond the package themselves". If I add Persistence to the paragraph, this "themselves" will be vague enough that it can mean "each other" I think.

Reply:
Works for me :smile: